<!doctype html>
<html lag="en">
  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <title>Publications | Herzik Lab</title>
  <!-- Botostrap 4.3 CSS -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
  <!-- Custom CSS -->
  <link rel="stylesheet" href="/assets/css/styles.css">
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700" rel="stylesheet">
  <!-- Favicons for every device type -->
  <link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicon/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicon/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicon/favicon-16x16.png">
  <link rel="manifest" href="/assets/img/favicon/site.webmanifest">
  <link rel="mask-icon" href="/assets/img/favicon/safari-pinned-tab.svg" color="#5bbad5">
  <meta name="msapplication-TileColor" content="#da532c">
  <meta name="theme-color" content="#ffffff">
  <link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Herzik Lab" />
  <!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Publications Herzik Lab | Herzik Lab</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Publications Herzik Lab" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Hello" />
<meta property="og:description" content="Hello" />
<link rel="canonical" href="http://localhost:4000/publications.html" />
<meta property="og:url" content="http://localhost:4000/publications.html" />
<meta property="og:site_name" content="Herzik Lab" />
<script type="application/ld+json">
{"url":"http://localhost:4000/publications.html","headline":"Publications Herzik Lab","description":"Hello","@type":"WebPage","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->

  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-134920918-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-134920918-1');
</script>


</head>

  <body>
    
<nav class="navbar navbar-expand-lg navbar-light">
  <div class="container">
    <a class="navbar-brand" href="/">
      <img src="/assets/img/herzik-lab-web-logo-1.svg" width="65" height="65" alt="herzik lab logo">
    </a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
      <ul class="navbar-nav">
        
        <li class="nav-item">
          <a href="/research.html" >Research</a>
        </li>
        
        <li class="nav-item">
          <a href="/publications.html" class="current" >Publications</a>
        </li>
        
        <li class="nav-item">
          <a href="/manual-plunge.html" >Manual-Plunge</a>
        </li>
        
        <li class="nav-item">
          <a href="/members.html" >Members</a>
        </li>
        
        <li class="nav-item">
          <a href="/join.html" >Join</a>
        </li>
        
        <li class="nav-item">
          <a href="/contact.html" >Contact</a>
        </li>
        
      </ul>
    </div>
  </div>
</nav>

    <div class="container">
      <div class="wrapper">
        <div class="container">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      <h1 class="page-title">Publications</h1>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>High-resolution structure determination of sub-100 kDa complexes using conventional cryo-EM. Nature Communications. Accepted. 2019</strong></h4><br />
      <div class="citation-spacing">Herzik MA. Jr.*, Wu M*, Lander GC.<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Determining high-resolution structures of biological macromolecules with masses of less than 100 kilodaltons (kDa) has long been a goal of the cryo-electron microscopy (cryo-EM) community. While the Volta Phase Plate has enabled cryo-EM structure determination of biological specimens of this size range, use of this instrumentation is not yet fully automated and can present technical challenges. Here, we show that conventional defocus-based cryo-EM methodologies can be used to determine the high-resolution structures of specimens amassing less than 100 kDa using a transmission electron microscope operating at 200 keV coupled with a direct electron detector. Our ~2.9 Å structure of alcohol dehydrogenase (82 kDa) proves that bound ligands can be resolved with high fidelity, indicating that these methodologies can be used to investigate the molecular details of drug-target interactions. Our ~2.8 Å and ~3.2 Å resolution structures of methemoglobin demonstrate that distinct conformational states can be identified within a dataset for proteins as small as 64 kDa. Furthermore, we provide the first sub-nanometer cryo-EM structure of a protein smaller than 50 kDa.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: Accepted</li>
          <li>PMCID: Accepted</li>  
            
          <li>    Biorxiv Preprint: <a href="https://www.biorxiv.org/content/10.1101/489898v1">489898</a></li>
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 6NBB, 6NBC, 6NBD </li>
          <li>EMDBs: EMD-0406, EMD-0407, EMD-0408 </li>
            
          <li>    Raw Data: EMPIAR-10249, EMPIAR-10250, EMPIAR-10252</li>
                 
            
          <li>Lab: <a href="http://www.lander-lab.com">Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
        <img src="/assets/img/publications/128561.png" class="img-responsive" /><br />
      
      <h4><strong>A Multi-model Approach to Assessing Local and Global Cryo-EM Map Quality. Structure. 2019</strong></h4><br />
      <div class="citation-spacing">Herzik MA Jr., Fraser. JS, Lander, GC<br />
       </div>
      
        <strong>Abstract</strong><br />
        There does not currently exist a standardized indicator of how well a cryo-EM-derived model represents the density from which it was generated. We present a straightforward methodology that utilizes freely available tools to generate a suite of independent models and to evaluate their convergence in an EM density. These analyses provide both a quantitative and qualitative assessment of the precision of the models and their representation of the density, respectively, while concurrently providing a platform for assessing both global and local EM map quality. We further use standardized datasets to provide an expected model-model agreement criterion for EM maps reported to be at 5 Å resolution or better. Associating multiple atomic models with a deposited EM map provides a rapid and accessible reporter of convergence, a strong indicator of highly resolved molecular detail, and is an important step toward an FSC-independent assessment of map and model quality.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 30449687</li>
          <li>PMCID: PMC6365196</li>  
            
          <li>    Biorxiv Preprint: <a href="https://www.biorxiv.org/content/10.1101/128561v1">128561</a></li>
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs:  </li>
          <li>EMDBs:  </li>
                 
            
          <li>Lab: <a href="">Fraser Lab, Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Conformational ensemble of the human TRPV3 ion channel. Nature Communications. 2018.</strong></h4><br />
      <div class="citation-spacing">Zubcevic L*, Herzik MA Jr*, Wu M*, Borschel WF, Hirschi M, Song AS, Lander GC, Lee SY.<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Transient receptor potential vanilloid channel 3 (TRPV3), a member of the thermosensitive TRP (thermoTRPV) channels, is activated by warm temperatures and serves as a key regulator of normal skin physiology through the release of pro-inflammatory messengers. Mutations in trpv3 have been identified as the cause of the congenital skin disorder, Olmsted syndrome. Unlike other members of the thermoTRPV channel family, TRPV3 sensitizes upon repeated simulation, yet a lack of structural information about the channel precludes a molecular-level understanding of TRPV3 sensitization and gating. Here, we present the cryo-electron microscopy structures of apo and sensitized human TRPV3, as well as several structures of TRPV3 in the presence of the common thermoTRPV agonist 2-aminoethoxydiphenyl borate (2-APB). Our results show α-to-Φ-helix transitions in the S6 during sensitization, and suggest a critical role for the S4-S5 linker Φ-helix during ligand gating.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 30429472</li>
          <li>PMCID: PMC6235889</li>  
            
          <li>    Biorxiv Preprint: <a href="https://www.biorxiv.org/content/10.1101/395616v1">395616</a></li>
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 6MHO, 6MHS, 6MHV, 6MHW, and 6MHX </li>
          <li>EMDBs: EMD-9115, EMD-9117, EMD-9119, EMD-9120, and EMD-9121 </li>
                 
            
          <li>Lab: <a href="http://www.lander-lab.com">Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Cryo-EM structure of a mitochondrial calcium uniporter. Science. 2018</strong></h4><br />
      <div class="citation-spacing">Yoo J*, Wu M*, Yin Y, Herzik MA Jr, Lander GC, Lee SY.<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Calcium transport plays an important role in regulating mitochondrial physiology and pathophysiology. The mitochondrial calcium uniporter (MCU) is a calcium-selective ion channel that is the primary mediator for calcium uptake into the mitochondrial matrix. Here, we present the cryo-electron microscopy structure of the full-length MCU from Neurospora crassa to an overall resolution of ~3.7 angstroms. Our structure reveals a tetrameric architecture, with the soluble and transmembrane domains adopting different symmetric arrangements within the channel. The conserved W-D-Φ-Φ-E-P-V-T-Y sequence motif of MCU pore forms a selectivity filter comprising two acidic rings separated by one helical turn along the central axis of the channel pore. The structure combined with mutagenesis gives insight into the basis of calcium recognition.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 29954988</li>
          <li>PMCID: PMC6155975</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 6DT0 </li>
          <li>EMDBs: EMD-8911 </li>
                 
            
          <li>Lab: <a href="http://www.lander-lab.com">Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Structural and cellular mechanisms of peptidyl-prolyl isomerase Pin1-mediated enhancement of Tissue Factor gene expression, protein half-life, and pro-coagulant activity. Haematologica. 2018</strong></h4><br />
      <div class="citation-spacing">Kurakula K, Koenis DS, Herzik MA Jr, Liu Y, Craft JW Jr, van Loenen PB, Vos M, Tran MK, Versteeg HH, Goumans MTH, Ruf W, de Vries CJM, Şen M.<br />
       </div>
      
        <strong>Abstract</strong><br />
        Tissue Factor is a cell-surface glycoprotein expressed in various cells of the vasculature and is the principal regulator of the blood coagulation cascade and hemostasis. Notably, aberrant expression of Tissue Factor is associated with cardiovascular pathologies such as atherosclerosis and thrombosis. Here, we sought to identify factors that regulate Tissue Factor gene expression and activity. Tissue Factor gene expression is regulated by various transcription factors, including activating protein-1 and nuclear factor-κ B. The peptidyl-prolyl isomerase Pin1 is known to modulate the activity of these two transcription factors, and we now show that Pin1 augments Tissue Factor gene expression in both vascular smooth muscle cells and activated endothelial cells via activating protein-1 and nuclear factor-κ B signaling. Furthermore, the cytoplasmic domain of Tissue Factor contains a well-conserved phospho-Ser258-Pro259 amino-acid motif recognized by Pin1. Using co-immunoprecipitation and solution nuclear magnetic resonance spectroscopy, we show that the WW-domain of Pin1 directly binds the cytoplasmic domain of Tissue Factor. This interaction occurs via the phospho-Ser258-Pro259 sequence in the Tissue Factor cytoplasmic domain and results in increased protein half-life and pro-coagulant activity. Taken together, our results establish Pin1 as an upstream regulator of Tissue Factor-mediated coagulation, thereby opening up new avenues for research into the use of specific Pin1 inhibitors for the treatment of diseases characterized by pathological coagulation, such as thrombosis and atherosclerosis.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 29545340</li>
          <li>PMCID: PMC6058786</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs:  </li>
          <li>EMDBs:  </li>
                 
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Cryo-electron microscopy structure of the lysosomal calcium-permeable channel TRPML3. Nature. 2017</strong></h4><br />
      <div class="citation-spacing">Hirschi M*, Herzik MA Jr*, Wie J, Suo Y, Borschel WF, Ren D, Lander GC, Lee SY.<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        The modulation of ion channel activity by lipids is increasingly recognized as a fundamental component of cellular signalling. The transient receptor potential mucolipin (TRPML) channel family belongs to the TRP superfamily and is composed of three members: TRPML1-TRPML3. TRPMLs are the major Ca2+-permeable channels on late endosomes and lysosomes (LEL). They regulate the release of Ca2+ from organelles, which is important for various physiological processes, including organelle trafficking and fusion. Loss-of-function mutations in the MCOLN1 gene, which encodes TRPML1, cause the neurodegenerative lysosomal storage disorder mucolipidosis type IV, and a gain-of-function mutation (Ala419Pro) in TRPML3 gives rise to the varitint-waddler (Va) mouse phenotype. Notably, TRPML channels are activated by the low-abundance and LEL-enriched signalling lipid phosphatidylinositol-3,5-bisphosphate (PtdIns(3,5)P2), whereas other phosphoinositides such as PtdIns(4,5)P2, which is enriched in plasma membranes, inhibit TRPMLs. Conserved basic residues at the N terminus of the channel are important for activation by PtdIns(3,5)P2 and inhibition by PtdIns(4,5)P2. However, owing to a lack of structural information, the mechanism by which TRPML channels recognize PtdIns(3,5)P2 and increase their Ca2+ conductance remains unclear. Here we present the cryo-electron microscopy (cryo-EM) structure of a full-length TRPML3 channel from the common marmoset (Callithrix jacchus) at an overall resolution of 2.9 Å. Our structure reveals not only the molecular basis of ion conduction but also the unique architecture of TRPMLs, wherein the voltage sensor-like domain is linked to the pore via a cytosolic domain that we term the mucolipin domain. Combined with functional studies, these data suggest that the mucolipin domain is responsible for PtdIns(3,5)P2 binding and subsequent channel activation, and that it acts as a 'gating pulley' for lipid-dependent TRPML gating.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 29019979</li>
          <li>PMCID: PMC5762132</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 5W3S </li>
          <li>EMDBs: EMD-8764 </li>
                 
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Achieving better-than-3-Å resolution by single-particle cryo-EM at 200 keV. Nature Methods. 2017</strong></h4><br />
      <div class="citation-spacing">Herzik MA Jr*, Wu M*, Lander GC<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Technical and methodological advances in single-particle cryo-electron microscopy (cryo-EM) have expanded the technique into a resolution regime that was previously only attainable by X-ray crystallography. Although single-particle cryo-EM has proven to be a useful technique for determining the structures of biomedically relevant molecules at near-atomic resolution, nearly 98% of the structures resolved to better than 4 Å resolution have been determined using 300 keV transmission electron microscopes (TEMs). We demonstrate that it is possible to obtain cryo-EM reconstructions of macromolecular complexes at a range of sizes to better than 3 Å resolution using a 200 keV TEM. These structures are of sufficient quality to unambiguously assign amino acid rotameric conformations and identify ordered water molecules, features previously thought only to be resolvable using TEMs operating at 300 keV.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 28991891</li>
          <li>PMCID: PMC5679434</li>  
            
          <li>    Biorxiv Preprint: <a href="https://www.biorxiv.org/content/10.1101/141994v1">141994</a></li>
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 5VY3, 5VY4, 5VY5 </li>
          <li>EMDBs: EMD-8741, EMD-8742, EMD-8743 </li>
            
          <li>    Raw Data: EMPIAR-10181, EMPIAR-10185, EMPIAR-10186</li>
                 
            
          <li>Lab: <a href="http://www.lander-lab.com">Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Regulation of nitric oxide signaling by formation of a distal receptor-ligand complex. Nature Chemical Biology. 2017</strong></h4><br />
      <div class="citation-spacing">Guo Y, Suess DLM, Herzik MA Jr, Iavarone AT, Britt RD, Marletta MA.<br />
       </div>
      
        <strong>Abstract</strong><br />
        The binding of nitric oxide (NO) to the heme cofactor of heme-nitric oxide/oxygen binding (H-NOX) proteins can lead to the dissociation of the heme-ligating histidine residue and yield a five-coordinate nitrosyl complex, an important step for NO-dependent signaling. In the five-coordinate nitrosyl complex, NO can reside on either the distal or proximal side of the heme, which could have a profound influence over the lifetime of the in vivo signal. To investigate this central molecular question, we characterized the Shewanella oneidensis H-NOX (So H-NOX)-NO complex biophysically under limiting and excess NO conditions. The results show that So H-NOX preferably forms a distal NO species with both limiting and excess NO. Therefore, signal strength and complex lifetime in vivo will be dictated by the dissociation rate of NO from the distal complex and the rebinding of the histidine ligand to the heme.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 28967923</li>
          <li>PMCID: PMC5698159</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs:  </li>
          <li>EMDBs:  </li>
                 
            
          <li>Lab: <a href="https://www.marlettalab.org/">Marletta Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Nitric Oxide-Induced Conformational Changes Govern H-NOX and Histidine Kinase Interaction and Regulation in Shewanella oneidensis. Biochemistry. 2017</strong></h4><br />
      <div class="citation-spacing">Rao M*, Herzik MA Jr*, Iavarone AT, Marletta MA<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Nitric oxide (NO) is implicated in biofilm regulation in several bacterial families via heme-nitric oxide/oxygen binding (H-NOX) protein signaling. Shewanella oneidensis H-NOX (So H-NOX) is associated with a histidine kinase (So HnoK) encoded on the same operon, and together they form a multicomponent signaling network whereby the NO-bound state of So H-NOX inhibits So HnoK autophosphorylation activity, affecting the phosphorylation state of three response regulators. Although the conformational changes of So H-NOX upon NO binding have been structurally characterized, the mechanism of HnoK inhibition by NO-bound So H-NOX remains unclear. In the present study, the molecular details of So H-NOX and So HnoK interaction and regulation are characterized. The N-terminal domain in So HnoK was determined to be the site of H-NOX interaction, and the binding interface on So H-NOX was identified using a combination of hydrogen-deuterium exchange mass spectrometry and surface-scanning mutagenesis. Binding kinetics measurements and analytical gel filtration revealed that NO-bound So H-NOX has a tighter affinity for So HnoK compared that of H-NOX in the unliganded state, correlating binding affinity with kinase inhibition. Kinase activity assays with binding-deficient H-NOX mutants further indicate that while formation of the H-NOX-HnoK complex is required for HnoK to be catalytically active, H-NOX conformational changes upon NO-binding are necessary for HnoK inhibition.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 28170222 </li>
          <li>PMCID: </li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs:  </li>
          <li>EMDBs:  </li>
                 
            
          <li>Lab: <a href="https://www.marlettalab.org/">Marletta Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Cryo-electron microscopy structure of the TRPV2 ion channel. Nature Structural and Molecular Biology. 2016</strong></h4><br />
      <div class="citation-spacing">Zubcevic L*, Herzik MA Jr*, Chung BC, Liu Z, Lander GC, Lee SY<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Transient receptor potential vanilloid (TRPV) cation channels are polymodal sensors involved in a variety of physiological processes. TRPV2, a member of the TRPV family, is regulated by temperature, by ligands, such as probenecid and cannabinoids, and by lipids. TRPV2 has been implicated in many biological functions, including somatosensation, osmosensation and innate immunity. Here we present the atomic model of rabbit TRPV2 in its putative desensitized state, as determined by cryo-EM at a nominal resolution of ∼4 Å. In the TRPV2 structure, the transmembrane segment 6 (S6), which is involved in gate opening, adopts a conformation different from the one observed in TRPV1. Structural comparisons of TRPV1 and TRPV2 indicate that a rotation of the ankyrin-repeat domain is coupled to pore opening via the TRP domain, and this pore opening can be modulated by rearrangements in the secondary structure of S6.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 26779611</li>
          <li>PMCID: PMC4876856</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 5AN8 </li>
          <li>EMDBs: EMD-6455 </li>
                 
            
          <li>Lab: <a href="https://www.lander-lab.com">Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Atomic structure of the 26S proteasome lid reveals the mechanism of deubiquitinase inhibition. Elife. 2016 Jan</strong></h4><br />
      <div class="citation-spacing">Dambacher CM*, Worden EJ*, Herzik MA Jr*, Martin A, Lander GC<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        The 26S proteasome is responsible for the selective, ATP-dependent degradation of polyubiquitinated cellular proteins. Removal of ubiquitin chains from targeted substrates at the proteasome is a prerequisite for substrate processing and is accomplished by Rpn11, a deubiquitinase within the 'lid' sub-complex. Prior to the lid's incorporation into the proteasome, Rpn11 deubiquitinase activity is inhibited to prevent unwarranted deubiquitination of polyubiquitinated proteins. Here we present the atomic model of the isolated lid sub-complex, as determined by cryo-electron microscopy at 3.5 Å resolution, revealing how Rpn11 is inhibited through its interaction with a neighboring lid subunit, Rpn5. Through mutagenesis of specific residues, we describe the network of interactions that are required to stabilize this inhibited state. These results provide significant insight into the intricate mechanisms of proteasome assembly, outlining the substantial conformational rearrangements that occur during incorporation of the lid into the 26S holoenzyme, which ultimately activates the deubiquitinase for substrate degradation.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 26744777</li>
          <li>PMCID: PMC4749569</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 3JCK </li>
          <li>EMDBs: EMD-6479 </li>
                 
            
          <li>Lab: <a href="https://www.lander-lab.com">Lander Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Structural insights into the role of iron-histidine bond cleavage in nitric oxide-induced activation of H-NOX gas sensor proteins. Proceedings of the National Academy of Sciences. 2014</strong></h4><br />
      <div class="citation-spacing">Herzik MA Jr, Jonnalagadda R, Kuriyan J, Marletta MA<br />
       </div>
      
        <strong>Abstract</strong><br />
        Heme-nitric oxide/oxygen (H-NOX) binding domains are a recently discovered family of heme-based gas sensor proteins that are conserved across eukaryotes and bacteria. Nitric oxide (NO) binding to the heme cofactor of H-NOX proteins has been implicated as a regulatory mechanism for processes ranging from vasodilation in mammals to communal behavior in bacteria. A key molecular event during NO-dependent activation of H-NOX proteins is rupture of the heme-histidine bond and formation of a five-coordinate nitrosyl complex. Although extensive biochemical studies have provided insight into the NO activation mechanism, precise molecular-level details have remained elusive. In the present study, high-resolution crystal structures of the H-NOX protein from Shewanella oneidensis in the unligated, intermediate six-coordinate and activated five-coordinate, NO-bound states are reported. From these structures, it is evident that several structural features in the heme pocket of the unligated protein function to maintain the heme distorted from planarity. NO-induced scission of the iron-histidine bond triggers structural rearrangements in the heme pocket that permit the heme to relax toward planarity, yielding the signaling-competent NO-bound conformation. Here, we also provide characterization of a nonheme metal coordination site occupied by zinc in an H-NOX protein.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 25253889</li>
          <li>PMCID: PMC4210026</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 4U99, 4U9B, 4U9G, 4U9J, 4U9K </li>
          <li>EMDBs: EMD-6479 </li>
                 
            
          <li>Lab: <a href="https://www.marlettalab.org/">Marletta Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Tunnels modulate ligand flux in a heme nitric oxide/oxygen binding (H-NOX) domain. Proceedings of the National Academy of Sciences. 2011</strong></h4><br />
      <div class="citation-spacing">Winter MB*, Herzik MA Jr*, Kuriyan J, Marletta MA<br />
      
        <div style="font-size: .8em; color: gray;">*These authors contributed equally</div>
       </div>
      
        <strong>Abstract</strong><br />
        Interior topological features, such as pockets and channels, have evolved in proteins to regulate biological functions by facilitating the diffusion of biomolecules. Decades of research using the globins as model heme proteins have clearly highlighted the importance of gas pockets around the heme in controlling the capture and release of O(2). However, much less is known about how ligand migration contributes to the diverse functions of other heme protein scaffolds. Heme nitric oxide/oxygen binding (H-NOX) domains are a conserved family of gas-sensing heme proteins with a divergent fold that are critical to numerous signaling pathways. Utilizing X-ray crystallography with xenon, a tunnel network has been shown to serve as a molecular pathway for ligand diffusion. Structure-guided mutagenesis results show that the tunnels have unexpected effects on gas-sensing properties in H-NOX domains. The findings provide insights on how the flux of biomolecules through protein scaffolds modulates protein chemistry.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 21997213</li>
          <li>PMCID: PMC3203760</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 3TF8, 3TF9, 3TFA, 3TFD, 3TFE, 3TFG, 3TF1 </li>
          <li>EMDBs:  </li>
                 
            
          <li>Lab: <a href="https://www.marlettalab.org/">Marletta Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Structure and properties of a bis-histidyl ligated globin from Caenorhabditis elegans. Biochemistry. 2010</strong></h4><br />
      <div class="citation-spacing">Yoon J, Herzik MA Jr, Winter MB, Tran R, Olea C Jr, Marletta MA<br />
       </div>
      
        <strong>Abstract</strong><br />
        Globins are heme-containing proteins that are best known for their roles in oxygen (O(2)) transport and storage. However, more diverse roles of globins in biology are being revealed, including gas and redox sensing. In the nematode Caenorhabditis elegans, 33 globin or globin-like genes were recently identified, some of which are known to be expressed in the sensory neurons of the worm and linked to O(2) sensing behavior. Here, we describe GLB-6, a novel globin-like protein expressed in the neurons of C. elegans. Recombinantly expressed full-length GLB-6 contains a heme site with spectral features that are similar to those of other bis-histidyl ligated globins, such as neuroglobin and cytoglobin. In contrast to these globins, however, ligands such as CO, NO, and CN(-) do not bind to the heme in GLB-6, demonstrating that the endogenous histidine ligands are likely very tightly coordinated. Additionally, GLB-6 exhibits rapid two-state autoxidation kinetics in the presence of physiological O(2) levels as well as a low redox potential (-193 +/- 2 mV). A high-resolution (1.40 A) crystal structure of the ferric form of the heme domain of GLB-6 confirms both the putative globin fold and bis-histidyl ligation and also demonstrates key structural features that can be correlated with the unusual ligand binding and redox properties exhibited by the full-length protein. Taken together, the biochemical properties of GLB-6 suggest that this neural protein would most likely serve as a physiological sensor for O(2) in C. elegans via redox signaling and/or electron transfer.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 20518498</li>
          <li>PMCID: PMC2903878</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 3MVC </li>
          <li>EMDBs:  </li>
                 
            
          <li>Lab: <a href="https://www.marlettalab.org/">Marletta Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Structural insights into the molecular mechanism of H-NOX activation. Protein Science. 2010</strong></h4><br />
      <div class="citation-spacing">Olea C Jr, Herzik MA Jr, Kuriyan J, Marletta MA<br />
       </div>
      
        <strong>Abstract</strong><br />
        Nitric oxide (NO) signaling in mammals controls important processes such as smooth muscle relaxation and neurotransmission by the activation of soluble guanylate cyclase (sGC). NO binding to the heme domain of sGC leads to dissociation of the iron-histidine (Fe-His) bond, which is required for enzyme activity. The heme domain of sGC belongs to a larger class of proteins called H-NOX (Heme-Nitric oxide/OXygen) binding domains. Previous crystallographic studies on H-NOX domains demonstrate a correlation between heme bending and protein conformation. It was unclear, however, whether these structural changes were important for signal transduction. Subsequent NMR solution structures of H-NOX proteins show a conformational change upon disconnection of the heme and proximal helix, similar to those observed in the crystallographic studies. The atomic details of these conformational changes, however, are lacking in the NMR structures especially at the heme pocket. Here, a high-resolution crystal structure of an H-NOX mutant mimicking a broken Fe-His bond is reported. This mutant exhibits specific changes in heme conformation and major N-terminal displacements relative to the wild-type H-NOX protein. Fe-His ligation is ubiquitous in all H-NOX domains, and therefore, the heme and protein conformational changes observed in this study are likely to occur throughout the H-NOX family when NO binding leads to rupture of the Fe-His bond.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 20162612</li>
          <li>PMCID: PMC2867026</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 3LAH, 3LAI </li>
          <li>EMDBs:  </li>
                 
            
          <li>Lab: <a href="https://www.marlettalab.org/">Marletta Lab</a></li>
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>

<hr style="padding-top: 1em;" />

<div class="container publication">
  <div class="row">
    <div class="col-md-2">
    </div>
    <div class="col-md-8">
      
      <h4><strong>Spectroscopic Characterization of Successive Phosphorylation of the Tissue Factor Cytoplasmic Region. Open Spectroscopy Journal. 2009</strong></h4><br />
      <div class="citation-spacing">Sen M, Herzik MA Jr, Craft JW, Creath AL, Agrawal S, Ruf W, Legge GB<br />
       </div>
      
        <strong>Abstract</strong><br />
        Tissue Factor (TF) is well known for its role during the activation of the coagulation pathway, but it is also critical for tumor biology and inflammation through protease activated receptor (PAR) 2 signaling. This signaling function is modulated by the successive phosphorylation of residues Ser253 and Ser258 within the TF cytoplasmic region (TFCR). This paper reports how we used NMR and spectroscopic methods to investigate the structural propensities of the unphosphorylated and phosphorylated forms of the TFCR. When unphosphorylated, the TFCR forms a local hydrophobic collapse around Trp254 and an electropositive patch from the membrane proximal basic block (Arg246-Lys247) to the conserved PKCalpha consensus residue Lys255. Phosphorylation of Ser253 alters the charge characteristics of this membrane proximal region, thereby strengthening the interaction between residue Ala248 and the Trp254 aromatic group. Phosphorylation of the Ser258-Pro259 motif destabilizes a turn at the C-terminus to form an extended polyproline helical motif. Our data suggests that by changing both its charge and local structural propensity, covalent modifications of the TFCR can potentially regulate its association with the cellular membrane and its signaling partners.
      
    </div>
    <div class="col-md-2">
    </div>
  </div>
  <div class="row" style="padding-top: 2em;">
    <div class="col-md-2">
    </div>
    <div class="col-md-4">
      <p><strong>Access the Paper</strong></p>
        <ul style="color: gray;">
          <li>PMID: 20076769</li>
          <li>PMCID: PMC2805860</li>  
            
        </ul>
    </div>
    <div class="col-md-4">
      <p><strong>Additional Links</strong></p>
        <ul style="color: gray">
          <li>PDBs: 2CEF 2CEH, 2CEZ, 2CFJ </li>
          <li>EMDBs:  </li>
                 
            
        </ul>
    </div>
    <div class="col-md-2">
    </div>
  </div>
</div>


      </div>
    </div>

    <!-- Bootstrap JavaScript plugins (must be in this order)-->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
  </body>
  <footer>
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <p class="footer-text">&copy; 2019 Herzik Lab & The University of California San Diego.</p>
      </div>
    </div>
  </div>
</footer>

</html>
